src/nativeresources/processrc.awk

```diff
@@ -54,12 +56,14 @@ BEGIN {
         gsub(/" +"/, "", $0);
         # remove all terminating newlines from the string - the msgfmt fails on those
         # since it expects them to be at the end of the msgid as well
-        while(gsub(/\\n"/, "\"", $0)) {}
+        if (removeTerm == 1) {
```

Comment:
Yes. The original contents of "/\n" should be retained. Otherwise all help messages are just messy -- one giant singleline when it is used for ildasm. I was actually questioning why we stripped this out but just wanted to keep the original semantic in mscorrc use.